### ISSUE TITLE
permission conflict when using multiple customviews

### ISSUE THREAD

**Issue by solverat** [pimcore/pimcore#730]
### reproduction
1.) create two custom views:

```php
<?php

return [
    "views" => [
        [
            "treetype" => "object",
            "name" => "CustomView1",
            "condition" => NULL,
            "id" => 1,
            "rootfolder" => "/customview-1",
            "showroot" => FALSE
        ],
        [
            "treetype" => "object",
            "name" => "CustomView2",
            "condition" => NULL,
            "id" => 2,
            "rootfolder" => "/customview-2",
            "showroot" => FALSE
        ]
    ]
];
```
2.) add "customview-1" and "customview-2" to the object tree root 
3.) add one or more objects (content can be anything, doesn't matter) to each tree
4.) create a user (allow documents, objects, assets)
5.) set object workspace like so (one with all rights, one with no rights):

![bildschirmfoto 2016-07-07 um 12 58 17](https://cloud.githubusercontent.com/assets/700119/16651264/220698a2-4443-11e6-9eec-fe97f256bbc9.png)

6.) save and log in with that user. 
7.) CustomView1 should be hidden and CustomView2 visible.
8.) try to delete / publish / unpublish an object from /customview-2
9. Javascript Error:

```javascript
Uncaught TypeError: Cannot read property 'getView' of undefined
```
 
![bildschirmfoto 2016-07-07 um 12 58 47](https://cloud.githubusercontent.com/assets/700119/16651346/a4b02c50-4443-11e6-80a0-c1b2e17c2e50.png)

**Comment by solverat** [comment]
@mrudnicki-pimcore i think you should reopen the ticket since this bug hasn't been fixed.

**Comment by mrudnicki-pimcore** [comment]
@solverat I fixed it in: 61cf463
Forgive, I wrote a bad commit message.

**Comment by mrudnicki-pimcore** [comment]
@solverat this one: 09900a5
